src/numberGenerator.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+const numberGenerator = () => {
+  const number = [];
+
+  while (number.length < 4) {
+    const digit = Math.floor(Math.random() * 9);
+
+    if (!number.includes(digit)) {
+      number.push(digit);
+    }
+  }
+
+  if (number[0] === 0) {
+    number[0] = 1;
+  };
```

Comment:
Imagine the situation: 

At this point our `number` is `[0,3,1,4]`, we do this check (`if (number[0] === 0) {...`) and **now** the `number` becomes `[1,3,1,4]`, which is incorrect because we have two "1".

Just check inside the `while` loop if we are pushing the first number (you can use `number.length === 0`) and that the new number is not "0".